velox/docs/functions/presto/binary.rst

```diff
@@ -58,6 +58,10 @@ Binary Functions
 
     Encodes ``binary`` into a base64 string representation.
 
+.. function:: to_base64url(binary) -> varchar
+
+    Encodes ``binary`` into a base64 string representation using the URL safe alphabet.
```

Comment:
Perhaps, turn "URL safe alphabet" into a link to https://www.rfc-editor.org/rfc/rfc4648#section-5 to provide some context for folks who are not familiar with it.